src/aui/auibar.cpp

```diff
@@ -366,11 +354,7 @@ void wxAuiGenericToolBarArt::DrawButton(
     dc.SetTextForeground(wxSystemSettings::GetColour(wxSYS_COLOUR_CAPTIONTEXT));
     if (item.GetState() & wxAUI_BUTTON_STATE_DISABLED)
     {
-#ifdef __WXMAC__
         dc.SetTextForeground(wxSystemSettings::GetColour(wxSYS_COLOUR_INACTIVECAPTIONTEXT));
-#else
-        dc.SetTextForeground(DISABLED_TEXT_COLOR);
```

Comment:
Disabled is not the same as inactive. There is no way to get the disabled color. If `wxSYS_COLOUR_INACTIVECAPTIONTEXT` should be the color of disabled, then that color should be changed.